src/style/badreset.css

```diff
@@ -0,0 +1,6 @@
+* {
+  font-family: "Roboto", "Segoe UI", sans-serif;
+  margin: 0;
+  padding: 0;
```

Comment:
don't reset margins and paddings using * selector
reset only needed properties for needed elements